src/PIL/ImageMode.py

```diff
@@ -52,6 +52,11 @@ def getmode(mode):
             "HSV": ("RGB", "L", ("H", "S", "V")),
             # extra experimental modes
             "RGBa": ("RGB", "L", ("R", "G", "B", "a")),
+            "BGR": ("BGR", "L", ("B", "G", "R")),
```

Comment:
I don't see the need for this? BGR is not listed in the docs or in [Storage.c](https://github.com/python-pillow/Pillow/blob/0648692b2c18cd42f5d20aef9ecf3d65c9f98941/src/libImaging/Storage.c#L131)

Reply:
@radarhere Sure, we can remove it. In this case, what should be the base mode for the respective `BGR;X`? That mode itself?

Reply:
You've already listed the base mode for `BGR;*` as just `RGB`.

Reply:
@radarhere Oh, right. That was unintentional on my part. Initially I thought the basemode for `BGR;*` should be `BGR` and added mode `BGR` correspondingly.

However, since `RGB` is the desired mode, I will keep it and drop mode `BGR`.